webhooks/cluster_test.go

```diff
@@ -19,6 +19,10 @@ package webhooks
 import (
 	"testing"
 
+	"sigs.k8s.io/cluster-api/internal/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
```

Comment:
nit: move down

We can compare Intellij versions/configs if you want. I'm not sure what solved the issue in my Intellij but I don't have the problem anymore since a few months.